lingua_franca/lang/parse_en.py

```diff
@@ -622,8 +621,12 @@ def extract_datetime_en(text, anchorDate=None, default_time=None):
     """
 
     def clean_string(s):
+        # normalize and lowercase utt  (replaces words with numbers=
+        s = normalize_en(s, remove_articles=False).lower()
+        # restore some decimal markers
+        s = s.replace("0.5", "half").replace("0.25", "quarter")
         # clean unneeded punctuation and capitalization among other things.
-        s = s.lower().replace('?', '').replace('.', '').replace(',', '') \
+        s = s.replace('?', '').replace('.', '').replace(',', '') \
             .replace(' the ', ' ').replace(' a ', ' ').replace(' an ', ' ') \
             .replace("o' clock", "o'clock").replace("o clock", "o'clock") \
             .replace("o ' clock", "o'clock").replace("o 'clock", "o'clock") \
```

Comment:
OOS here, but surely this could be a regex looking for (invalid regex) `o*'*clock` (noting here because I just noticed)

Reply:
out of scope, will be handled once my datetime rewrite is finished, essentially any changes to extract_datetime are currently a waste of time